components/talksPage/talksPage.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import style from './talksPage.module.scss';
+import talksList from '../../assets/videos.json';
+import YouTube from 'react-youtube';
+
+function TalksPage() {
+  const [currSrc, setSrc] = useState('GHbNaDSWUX8');
+
+  const calcArrowSide = (curr, side, length) => {
+    if (side > curr) {
+      if (curr === length) {
+        return 1;
+      }
+    } else if (side < curr) {
+      if (curr === 1) {
+        return length;
+      }
+    }
+    return side;
+  };
+
+  const onVideoClick = (event) => {
+    event.target.pauseVideo(); // stop chosen video
+    setSrc(event.target.playerInfo.videoData.video_id); // open the chosen video on the big iframe
+  };
+
+  const setVideos = (videos) => { // add youtube videos
+    const opts = {
```

Comment:
create consts after imports

Reply:
This const is only relevant to this method so I decided to leave it as it is